hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1021,6 +1021,49 @@ function upgrade_1023() {
     return true;
   }
 
+  /**
+   * Creates Option Group and Option Values for Health an Life Insurance Plan 
+   * Types.
+   * 
+   * @return boolean TRUE
+   */
+  public function upgrade_1025() {
```

Comment:
also can you rename the method to upgrade_1024 since there is no upgrader here with that number instead of jumping to 1025